docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -440,6 +441,7 @@ Json Parameters:
           `Ulimits: { "Name": "nofile", "Soft": 1024, "Hard": 2048 }`
     -   **SecurityOpt**: A list of string values to customize labels for MLS
         systems, such as SELinux.
+    -   **StorageOpt**: Storage driver options per container
```

Comment:
I realize some other options are missing this information, but can we add a simple description of the format in which these options should be passed (similar to "LogConfig" below), I.e. Is it `["foo=bar", "bar=baz"]` or `[{"foo":"bar"}]`?